Server/Components/Pawn/Script/Script.cpp

```diff
@@ -12,6 +12,7 @@
 */
 
 #define _Static_assert static_assert
+#define STKMARGIN ((cell)(16 * sizeof(cell))) // from amx.c
```

Comment:
Wouldn't this be better exposed through a function or common header so if it's ever changed it's consistent?